### ISSUE TITLE
[A11y_Bug]: Content overlapped or cut off in browser extension

### ISSUE THREAD

**Issue by dankeck** [IBMa/equal-access#1161]
### Project

a11y checker extension

### Browser

Firefox

### Operating System

Windows

### Assistive technology

Increased font size

### Description

Please see the following screenshots where components of the Firefox extension do not reflow correctly when they run out of room. 

1. Scan button cannot be clicked because it is behind another control. ![image](https://user-images.githubusercontent.com/86371013/202240295-ef3d1cc5-c16c-4d89-a759-542bd2169cd7.png)

2. Text is cut off and there is no horizontal scrollbar. ![image](https://user-images.githubusercontent.com/86371013/202240462-1bff3c51-19e8-4915-97c1-cdde094c8e00.png)

3. Text bleeds into adjoining frame. ![image](https://user-images.githubusercontent.com/86371013/202240530-64c76e18-7afb-433d-a5ec-3a5b33ce16e9.png)

Of course this all works fine on a setup with high resolution and plenty of screen real estate, which is the ideal setup. But sometimes I work on a laptop, and I always have increased font size due to having trouble seeing small text. A workaround is to temporarily make frames larger until the button is clickable and text is readable. But it would be great if content flowed onto the next line, or at least a horizontal scrollbar were available. 

### Steps to reproduce

- Open Accessibility Assessment or Accessibility Checker tab in Developer Tools.
- Resize browser window to be sufficiently small, and/or increase text size to be sufficiently large.

**Comment by aliunwala** [comment]
Triage:
We are also aware of this and have seen this issue from time to time. We hope to try update our carbon grid to deal with boundaries so reactive reflow ends up not jumping out of the grids. 

We will add this to our work backlog

**Comment by aliunwala** [comment]
@tombrunet Fyi for this ticket ^

**Comment by tombrunet** [comment]
This is fixed in branch [2023-ext-arch](https://github.com/IBMa/equal-access/tree/2023-ext-arch). You'll see it in the next major update for the extension. Coming soon.
